main.py

```diff
@@ -115,25 +143,43 @@ def main():
     parser.add_argument('--since', dest='since_id', help='ID after which to import')
     parser.add_argument('--url', dest='url_only', help='only output URL', action='store_const', const=True, default=False)
     parser.add_argument('--csv', dest='csv_only', help='only output CSV', action='store_const', const=True, default=False)
+    parser.add_argument('--count', dest='count_only', help='only output count', action='store_const', const=True, default=False)
+
```

Comment:
nice, running the --count on the 28th is showing the total amount for that day that we found